### ISSUE TITLE
Linking error while including <cuda/pipeline> in 2 .cu files

### ISSUE THREAD

**Issue by JustLuoyu** [NVIDIA/cuda-samples#94]
multiple definition of `cuda::__3::pipeline<(cuda::__3::thread_scope)2>::__barrier_try_wait_parity(cuda::__3::barrier<(cuda::__3::thread_scope)2, cuda::std::__3::__empty_completion>&, bool)'

**Comment by value56** [comment]
Hi,did you solve this problem?

**Comment by JustLuoyu** [comment]
nope, I think this feature is not ready to use yet. I've met more issues.

**Comment by rwarmstr** [comment]
Given the age of this issue will close as resolved.
